explainer.md

```diff
@@ -368,6 +368,16 @@ if (await draftsBucket.durability() !== "strict") {
 
 A bucket's durability policy cannot be changed once the bucket is created.
 
+## Storage policy: Max Quota 
+
+A bucket's max quota policy allows developers to specify the maximum amount of storage a bucket is allowed to use. This helps with quota management by ensuring a bucket does not use storage meant for other higher priority buckets. 
```

Comment:
Your explanation is correct but subtle.

How about saying that per-bucket quotas can be used to place an upper bound on storage usage for each application feature, so that a bug in an application feature doesn't impact the other features' ability to store data by eating up the entire origin's quota?